third_party/move/move-compiler-v2/tests/checking/typing/vector_with_non_base_type_inferred_2.move

```diff
@@ -0,0 +1,32 @@
+module 0x42::Test {
```

Comment:
This test is derived from vector_with_non_base_type_inferred to reveal the complete error message from V1.

Reply:
Again the same bug.